modules/roles_profiles/manifests/profiles/openssh.pp

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::openssh {
```

Comment:
move all of this to it's own windows specific module.  win_opensshd

Reply:
done